### PR TITLE
Update integration tests, add setInterval

### PR DESCRIPTION
**Pull request by ryscheng**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,7 @@ lib-cov
 *.out
 *.pid
 *.gz
+.*.swp
```
Comment (willscott) [inline .gitignore]: .files shouldn't be picked up by default?

File: lib/modulecontext.js [hunk 2]
```diff
@@ -20,6 +20,9 @@ var context = {
   setTimeout: function (cb, time) {
     setTimeout(cb, time);
   },
+  setInterval: function (cb, time) {
```
Comment (willscott) [inline lib/modulecontext.js]: add clearInterval as well

**Comment by willscott** [comment]
couple minor things, then good to merge.

File: lib/modulecontext.js (hunk 2 above)
Comment (ryscheng) [inline lib/modulecontext.js]: done

**Pull request merged**